Prime.cpp

```diff
@@ -0,0 +1,25 @@
+//This Code all prints all the prime number till input n 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `c/cpp spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp25unf0ja/Prime.cpp
+++ b/tmp/tmp25unf0ja/Prime.cpp
@@ -1,4 +1,4 @@
-//This Code all prints all the prime number till input n 
+//This Code all prints all the prime number till input n
 
 #include <bits/stdc++.h>
 using namespace std;
```